doped/utils/stenciling.py

```
@@ -0,0 +1,1122 @@
+from doped.utils.supercells import _largest_cube_length_from_matrix, min_dist
+from doped.utils.symmetry import (
+    SymmOp,
+    apply_symm_op_to_site,
+    apply_symm_op_to_struct,
+    get_clean_structure,
+    get_sga,
+    translate_structure,
+)
+
+
+def get_defect_in_supercell(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)